ceopardy.py

```diff
@@ -15,15 +15,28 @@
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 #
+import flask
 from flask import Flask, render_template
+from flask_socketio import SocketIO, send, emit
 from ceopardy.controller import Controller
+from ceopardy.api import api
+import sys
+import re
+import random
```

Comment:
PEP8: Order imports by name and with distinct groups separated by a newline: stdlib, external libs and project imports.

Just a saying for the future. Don't actually fix it :smile: